eldritch/events.py

```diff
@@ -356,8 +359,9 @@ def resolve(self, state):
     state.event_stack.append(self)
     self.stack_cleared = True
 
-    # TODO: lose half the items and clues.
```

Comment:
You're going to get some merge conflicts. Sorry. You might not get them if you revert your changes to Insane.

Reply:
Not as bad as it seemed...